BLSQ/iaso
# IA-3628, IA-3629 HOTFIX: org unit types and groups timeout bug

slow DB queries for groups and org unit types dropdowns in Change request pages

Related JIRA tickets : IA-628, IA-3629

## Self proofreading checklist


## Doc


## Changes

- Add `/api/v2/orgunittypes/dropdown` endpoint with only fields required by front 
- Disable group filter in org unit chnage requests page

## How to test

With DB with multiple data sources and lots(!) of org unit types and groups:

- Go to change requests list view
- Groups filter should be disabled
- Org Unit types dropdown should load reasonably fast

## Print screen / video

Org unit types fropdown request:

Before

![Screenshot 2024-10-31 at 10 25 44](https://github.com/user-attachments/assets/595fa558-1a5e-4e55-a6b3-d7bd9b1b58b6)

After

![Screenshot 2024-10-31 at 10 26 02](https://github.com/user-attachments/assets/19dbdb94-e55a-402a-aba3-603427c8e639)

change requests page

![Screenshot 2024-10-31 at 10 32 21](https://github.com/user-attachments/assets/10255491-8aca-4d64-a5aa-5fdda1b7105d)

## Notes

THIS HOTFIX IS FOR PLAYGROUND ONLY:

- DO NOT MERGE ON MAIN
- DO NOT MAKE FURTHER HOTFIXES FROM THIS BRANCH --> Use v1.246b. instead 

Migration conflicts are expected since this branch is behind main
